app/git/models.py

```diff
@@ -0,0 +1,73 @@
+
+
+# -*- coding: utf-8 -*-
+"""Define models.
+
+Copyright (C) 2021 Gitcoin Core
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU Affero General Public License as published
+by the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Affero General Public License for more details.
+
+You should have received a copy of the GNU Affero General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""
+import logging
+
+from django.db import models
+
+from economy.models import SuperModel
+
+logger = logging.getLogger(__name__)
+
+
+class GitCache(SuperModel):
```

Comment:
@nutrina can you add tests for this model?

Reply:
@ksolo Yes, I am on it